packages/@react-spectrum/s2/src/SegmentedControl.tsx

```diff
@@ -55,12 +56,14 @@ export const SegmentedControlContext = createContext<ContextValue<SegmentedContr
 const segmentedControl = style<{size: string}>({
   font: 'control',
   display: 'flex',
+  gap: 4,
   backgroundColor: 'gray-100',
-  borderRadius: 'lg',
-  width: 'full'
+  borderRadius: 'default',
+  width: 'fit'
```

Comment:
SegmentedControl now fits its contents by default rather than filling its parent

Reply:
Did spectrum say anything about this in the meeting?

Reply:
They were ok with the change to fit the content width by default. I added an `isJustified` prop to SementedControl to allow the previous behavior as an option, which also matches the same option in ActionButtonGroup and ToggleButtonGroup.